### ISSUE TITLE
About output

### ISSUE THREAD

**Issue by dengchengxifrank** [alexgo1/Pytorch-3D-R2N2#9]
Thank you for your great work!When I use torch.randn(3,3,128,128) as input,I get torch.Size([3, 2, 32, 32, 32]) as output,and I wonder why the second parameter is 2 instead of 1.Thank you!

**Comment by alex-golts** [comment]
Hi, thanks for using my code!
I guess it's a matter of definition, in my case I chose to have the decoder return 2 logits and do a softmax for converting them to probabilities. you're right that in the case of a binary classification, it is not required, and could have been solved with a single neuron, as the probability for one class is 1 minus the probability for the other. 
I hope this helps clarify it

**Comment by dengchengxifrank** [comment]
thank you！
